agent/taskresource/volume/dockervolume.go

```diff
@@ -152,16 +155,52 @@ func (cfg *EFSVolumeConfig) Source() string {
 	return cfg.DockerVolumeName
 }
 
+func (cfg *EFSVolumeConfig) GetType() string {
+	return EFSVolumeType
+}
+
+func (cfg *EFSVolumeConfig) GetVolumeId() string {
+	return ""
+}
+
+func (cfg *EFSVolumeConfig) GetVolumeName() string {
+	return cfg.DockerVolumeName
+}
+
 // Source returns the name of the volume resource which is used as the source of the volume mount
 func (cfg *DockerVolumeConfig) Source() string {
 	return cfg.DockerVolumeName
 }
 
+func (cfg *DockerVolumeConfig) GetType() string {
+	return DockerVolumeType
+}
+
+func (cfg *DockerVolumeConfig) GetVolumeId() string {
+	return ""
```

Comment:
ditto here -- is there a unique identifier we should return for a docker volume besides its name (which I believe is unique per host)?

Reply:
^ same response as above

Reply:
I assume the docker volume name?